test/legacy_allowlist.out

```diff
@@ -63,6 +63,10 @@
 0x00000618 0x0000000000ffffff # DRAM_POWER_LIMIT
 0x00000620 0x0000000000007f7f # UNCORE_RATIO_LIMIT
 0x00000621 0x0000000000000000 # UNCORE_RATIO_STATUS
+0x00000c8d 0x000003FF000000FF # IA32_QM_EVTSEL
+0x00000c8e 0x0000000000000000 # IA32_QM_CTR
+0x00000c8f 0xFFFFFFFF000001FF # IA32_PQR_ASSOC
+0x00000e00 0x00000000ffffffff # C0_MSR_PMON_BOX_CTL
```

Comment:
Additional testing may be needed on these bit fields, the msr_data_skx json file and the MSRIOGroup.cpp checks of bit width.  
The general support is:
![image](https://user-images.githubusercontent.com/23505009/120856605-f556a680-c534-11eb-8c45-71e5f5c8cfc9.png)

but E5 v4 shows:
![image](https://user-images.githubusercontent.com/23505009/120856774-2e8f1680-c535-11eb-98fc-fb8a103aa848.png)

E5 v4 support in the SDM is listed as: